Wangscape/tilegen/TilesetGenerator.h

```diff
@@ -1,24 +1,32 @@
 #pragma once
 #include <SFML/Graphics.hpp>
 #include <memory>
-#include "Options.h"
+#include "../Options.h"
```

Comment:
Oh, yes, sorry. I looked at `..` and I thought that you're going up to some `Wangscape` subdirectory.

So leave `"Options.h"` here, if you agree with me about include paths - I think that's enough to add `Wangscape` subdirectory as include directories and use relative paths starting from there. It makes clear, which file is included.

I've checked it and I made a little "design mistake" with `CMakeLists.txt`: currently `Wangscape/metaoutput/CMakeLists.txt` has
```
target_include_directories(metaoutput PUBLIC ${PROJECT_SOURCE_DIR}/Wangscape)
```
line, which implicitly adds `Wangscape` to include directories of `Wangscape` target.

I think that should be changed (I mean adding 
```
target_include_directories(Wangscape PUBLIC ${PROJECT_SOURCE_DIR}/Wangscape)
```
(could be probably `PRIVATE`) line to the main `CMakeLists.txt`) to make it obvious.